requirements/default.txt

```diff
@@ -1,7 +1,6 @@
 celery>=4.3.0,<5.0.0; python_version>="3.7"
 vine==1.3.0
```

Comment:
Shouldn't this one have been removed as well?

Reply:
Yes please? Still getting `flower 0.9.7 has requirement vine==1.3.0, but you'll have vine 5.0.0 which is incompatible.`.